### PR TITLE
Todo list - All files copied from old branch

### PR DESCRIPTION
**Pull request by learnwithalfred**
I just copied all the files from the previous branch. 
This is because you have to check if I am following the best practices

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      type="image/png"
+      href="https://upload.wikimedia.org/wikipedia/commons/thumb/5/53/Wikimedia-logo.png/480px-Wikimedia-logo.png"
+    />
+    <title class="text-center my-3">Sneak peek: To-do list</title>
+  <script defer src="/runtime.975caff4.js"></script><script defer src="/main.975caff4.js"></script></head>
+  <body>
+    <h1 class="text-center my-4">Sneak peek: To-do list</h1>
+    <div class="card">
+      <div class="card-header">Todo <span class="todoLength"></span></div>
+      <div class="input-group">
```
Comment (dqarias) [inline dist/index.html:21]: Kindly insteof using `<div>` labels you can replace them using `<section>, <article>, <form>` labels for use a semantical html

File: src/functions.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    return selectedTask;
+  }
+  return array;
+};
+
+const handleClearCompleted = (array) => {
+  const newTodoItems = array.filter((item) => {
+    if (!item.complete) return true;
+    return null;
+  });
```
Comment (dqarias) [inline src/functions.js]: Kindly try to avoid negative conditions for maintain javascript best practices

**Review by dqarias — CHANGES_REQUESTED** [review]
## Hi @learnwithalfred 👋,
Good job so far!

## To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️
[DRY, KISS, and YAGNI](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md) rules are followed ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

File: dist/index.html (hunk 1 above)
Comment (learnwithalfred) [reply dist/index.html:21]: Great

I actually want to put all that content in the section tag

File: src/functions.js (hunk 2 above)
Comment (learnwithalfred) [reply src/functions.js]: Issue has been resolved

thanks

**Review by dqarias — APPROVED** [review]
## Hi @learnwithalfred  👋
Your project is complete! ✔️ 🏁

Your code is very professional 💯. There is nothing else to say other than... it's time to merge it :shipit:

## Highlights 🌟
Descriptive commit messages ✔️
Descriptive PR title and description ✔️
No linter errors. ✔️
GitHub flow is followed. ✔️
[DRY, KISS, and YAGNI](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md) rules are followed ✔️
HTML, CSS, JavaScript best practices are followed ✔️

Cheers and Happy coding!👏👏👏

**Pull request merged**